autogpt/commands.py

```diff
@@ -162,8 +163,8 @@ def google_official_search(query, num_results=8):
     # Return the list of search result URLs
     return search_results_links
 
-def browse_website(url, question):
-    summary = get_text_summary(url, question)
+def browse_website(url):
```

Comment:
Again, this is removing another really powerful feature, and it's not even mentioned in the PR description or title.